apps/nextjs-website/react-components/components/Cards/Cards.tsx

```diff
@@ -199,8 +201,13 @@ const Cards = ({
           )}
         </>
       </Box>
+      {isNone && bottomCTA && (
```

Comment:
```suggestion
      {bottomCTA && (
```
I would allow the bottomCTA for any layout, not just 'none'